data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_domain/Range.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_impl.remedial_action.range_domain;
+
+import com.powsybl.iidm.network.Network;
+
+/**
+ * Range.
+ *
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+public interface Range {
+
+    public double getMinValue(Network network);
+
+    public double getMaxValue(Network network);
```

Comment:
Public modifiers are redundant, so you can remove them. Same comment for every interface.